app/views/users/feed.html.erb

```diff
@@ -0,0 +1,53 @@
+<% if @contents.length == 0 %>
+ <h3>Use the illuminate search bar above to begin!</h3>
+<% end %>
+
+<script sync src="https://platform.twitter.com/widgets.js"></script>
+
+<script>
+    window.onload = (function(){
+      $("#waiting").hide();
+      $("#feed").show();
+    });
+
+</script>
+
+<div id="waiting">
+  <div id="floatingCirclesG">
+  	<div class="f_circleG" id="frotateG_01"></div>
+  	<div class="f_circleG" id="frotateG_02"></div>
+  	<div class="f_circleG" id="frotateG_03"></div>
+  	<div class="f_circleG" id="frotateG_04"></div>
+  	<div class="f_circleG" id="frotateG_05"></div>
+  	<div class="f_circleG" id="frotateG_06"></div>
+  	<div class="f_circleG" id="frotateG_07"></div>
+  	<div class="f_circleG" id="frotateG_08"></div>
+  </div>
+</div>
+
+<div id="feed" style="display:none">
+  <% @contents.each do |content| %>
+    <% if content.provider == "vimeo" %>
```

Comment:
I think you use these kinds of checks in a few places. To DRY up your code, you might consider adding a `twitter?` method that will do this check for you. Then if the provider string ever changes, you can easily update it everywhere.